main/src/ca/uwaterloo/flix/util/Formatter.scala

```diff
@@ -131,33 +208,71 @@ object Formatter {
 
     override def black(s: String): String = Console.BLACK + s + Console.RESET
 
+    override def brightBlack(s: String): String = fgColor(128, 128, 128, s)
```

Comment:
@magnus-madsen  Should the colors be added to AnsiColor.scala instead of using rgb?

Reply:
What do you mean?

Reply:
Like for black, where I can write:
```
  override def black(s: String): String = Console.BLACK + s + Console.RESET
```
The `Console.BLACK` part.

Reply:
Oh. I would avoid using Console and just use rgb-- for consistency.

Reply:
Should I change the ones using `Console` to `rgb`? Or just leave them as is

Reply:
Change them to make it consistent